docs/README.md

```diff
@@ -113,49 +113,23 @@ To run the measurements, follow the below steps
 
 1. Open `<MeasurementName>.lvproj` that contains the Measurement Service library.
 
-2. Start the discovery service if the measurement is to be used in InstrumentStudio Measurement Framework.
+2. Start the discovery service if the measurement is to be used in InstrumentStudio.
 
 3. Run the `Run Service.vi` from `<MeasurementName>.lvlib` to run the measurement service from `<MeasurementName>.lvproj`
 
 4. To manually stop the service, click on the `Stop` button from the front panel of `Run Service.vi`.
 
 ---
 
-## Example Measurements
+## Examples
 
-The example measurements provided contain the following projects:
-
-- **LDO measurement**: The project performs simple LDO measurements. The type of LDO measurements performed are:
-  - Line Regulation
-  - Load Regulation
-
-### Dependencies for the example measurements
-
-1. LabVIEW 2020 64 bit
-2. NI-DC Power driver.
-3. NI-DMM driver.
-
-### Running the LDO measurement example
-
-To run the measurements, follow the steps mentioned below:
-
-1. Open `LDOMeasurements.lvproj` from the examples provided.
-
-2. Start the discovery service if the measurement is to be used in InstrumentStudio Measurement Framework.
-
-3. Run the following VI to run the measurement service from `LDOMeasurements.lvproj`
-    -To run Line Regulation, run `Run Service.vi` from `LineRegulation.lvlib`
-    -To run Load Regulation, run `Run Service.vi` from `LoadRegulation.lvlib`
-
-4. Use the `Measurement Framework` and run the measurements interactively in InstrumentStudio. For more details, please refer to the section [Using the measurement in InstrumentStudio](#using-the-measurement-in-instrumentstudio).
-
-Note: The Line and Load regulation measurements have not been tested with real hardware. They have only been tested with simulated instruments.
+The `Source\Example Measurements` directory contains example measurement services. See the [README.md](https://github.com/ni/measurementlink-labview/blob/main/Source/Example%20Measurements/README.md) file there for more information.
```

Comment:
If this markdown is only served up from GitHub, you can use relative paths.

When it's uploaded to PyPI or some other service, you can't.

Reply:
Using a relative path.